pkg/resources/kafka/kafka.go

```diff
@@ -905,6 +911,30 @@ func (r *Reconciler) createExternalListenerStatuses(log logr.Logger) (map[string
 	return extListenerStatuses, nil
 }
 
+func (r *Reconciler) getK8sAssignedNodeport(log logr.Logger, eListenerName string, brokerId int32) (int32, error) {
+	log.Info("determining automatically assigned nodeport",
+		"brokerId", brokerId, "listenerName", eListenerName)
+	nodePortSvc := &corev1.Service{}
+	err := r.Client.Get(context.Background(),
+		types.NamespacedName{Name: fmt.Sprintf("%s-%d-%s", r.KafkaCluster.GetName(), brokerId, eListenerName),
+			Namespace: r.KafkaCluster.GetNamespace()}, nodePortSvc)
+	if err != nil {
+		return 0, errors.WrapWithDetails(err, "could not get nodeport service",
+			"brokerId", brokerId, "listenerName", eListenerName)
+	}
+	log.Info("nodeport service successfully found",
```

Comment:
Do we really need this at Info level? This way we can print `[#external listeners]x[#ingress config]x[#brokers]` log entries worst case, which can pretty high even in a mid size cluster.
```suggestion
	log.V(1).Info("nodeport service successfully found",
```